### PR TITLE
PTV-1535 Added reference chains to client calls

### PR DESCRIPTION
**Pull request by bio-boris**
* Add reference chaining to avoid workspace permissions error
* Ran Black Formatter

### CODE REVIEW

File: lib/kb_stringtie/Utils/StringTieUtil.py [hunk 1]
```diff
@@ -205,36 +228,40 @@ def _get_gtf_file(self, alignment_ref, result_directory):
...
-                    genome_ref))
+                    genome_ref
+                )
+            )
         print(contig_id)
         log("Generating GFF file from Genome")
         try:
-            ret = self.au.get_assembly_as_fasta({'ref': contig_id})
-            fa_output_file = ret['path']
+            ret = self.au.get_assembly_as_fasta({"ref": genome_ref + ";" + contig_id})
```
Comment (bio-boris) [inline lib/kb_stringtie/Utils/StringTieUtil.py:250]: The actual change as you suggested in the ticket

**Comment by jayrbolton** [comment]
@bio-boris Were you able to run much testing here, manual or otherwise, and how did it go?

**Comment by bio-boris** [comment]
@jayrbolton I deployed it as a beta app and am running a job. It passed the initial point where it failed the reference chain, otherwise it is still running

**Review by jayrbolton — APPROVED** [review]
The ref chain fix looks good; did not read through all the automatic syntax changes

**Pull request merged**